### ISSUE TITLE
[BUG] [1.19.2] Aethersteel Bookshelf doesn't actually contribute to Apotheosis stats.

### ISSUE THREAD

**Issue by togera13** [Ross2750/Aethersteel#6]
**Version**
Forge Version _(43.2.5)_
Mod Version _(5.1)_

**How to reproduce**
Place the bookshelves around the Apotheosis enhanced enchanting table.
Look at the values.

**Expected behavior**
To have the Bookshelves effect match its tooltip.

**Screenshots**
(Optional)

I don't know if this is somehow a mod conflict, but it isn't working.

**Comment by Ross2750** [comment]
I think this issue happen because Aethersteel does not have compatibility on Apotheosis

**Comment by togera13** [comment]
> I think this issue happen because Aethersteel does not have compatibility on Apotheosis

Ah, maybe it has something to do with iron bookshelves having support for Apotheosis & somehow the game thinks Aethersteel bookshelves should have quanta but doesn't actually give it the effect.

TL;DR I tested it without the other mods and it is as you say it doesn't have compatibility.

If you could add compatibility for it at some point that would be cool though.
